src/components/Pagination/PageButton/PageButton.js

```diff
@@ -1,26 +1,53 @@
 import styled, { css } from 'react-emotion';
 
 import Button from '../../Button';
+import { calculatePadding } from '../../Button/components/RegularButton/RegularButton'; // eslint-disable-line max-len
 
 const baseStyles = ({ theme }) => css`
   label: pagination__buttom;
   border-radius: 0;
   border-right: 0;
+  font-weight: normal;
```

Comment:
There's a variable for that 😉

```suggestion
  font-weight: ${theme.fontWeight.regular};
```